lib/travis/config/defaults.rb

```diff
@@ -24,9 +24,10 @@ def logs_api_auth_token
     }
 
     define  host:                 'travis-ci.org',
+            api_com_url:          'https://api.travis-ci.com',
             shorten_host:         'trvs.io',
             public_mode:          !!ENV['PUBLIC_MODE'],
-            applications:         {},
+            applications:         { api_org: { token: 'sometoken', full_access: true } },
```

Comment:
What does `full_access` here mean?

Reply:
@aakritigupta, it's for access control logic. I did it similarly to admin and sync apps.

I'll remove it from here, it should be present only in heroku config vars.